File: 2_fizzbuzz/test_fizzbuzz.rb
require 'minitest/autorun'
require './fizzbuzz.rb'

class TestfizzBuzz < MiniTest::Unit::TestCase
  def setup
    @fizz_buzz = FizzBuzz.new
  end

  def test_fizzbuzz
    assert_equal 1, @fizz_buzz.fizzbuzz(1)
    assert_equal "Fizz", @fizz_buzz.fizzbuzz(3)
    assert_equal "Buzz", @fizz_buzz.fizzbuzz(5)
    assert_equal "FizzBuzz", @fizz_buzz.fizzbuzz(15)
  end
end

File: 2_fizzbuzz/test_fizzbuzz.py
import unittest
import fizzbuzz

class TestGizzBuzz(unittest.TestCase):

  def test_fizzbuzz(self):
    self.assertEqual('Fizz',fizzbuzz.fizzbuzz(3))
    self.assertEqual('Buzz',fizzbuzz.fizzbuzz(5))
    self.assertEqual('FizzBuzz',fizzbuzz.fizzbuzz(15))
    self.assertEqual(1,fizzbuzz.fizzbuzz(1))

if __name__ == "__main__":
      unittest.main()
